### PR TITLE
"Add voxpupuli forge account to credentials"

### PR DESCRIPTION
**Pull request by nibalizer**

### CODE REVIEW

File: bin/encrypt_credentials.sh [hunk 1]
```diff
@@ -1,14 +1,15 @@
 #!/bin/bash
 
-set -eu
+set -e
```
Comment (igalic) [inline bin/encrypt_credentials.sh]: why

**Comment by roidelapluie** [comment]
why "" in title of the pr?

**Comment by bastelfreak** [comment]
Thats the identifier that the commit is from @nibalizer (at least thats what I use the quotes for :D). I think the quotes are made by his bash/git setup?

**Comment by nibalizer** [comment]
https://github.com/nibalizer/dotfiles/blob/master/local/bin/git-pull-request is how I make my pull requests

**Comment by bastelfreak** [comment]
We need to reencrypt again. So I will merge + provide a PR to fix.

**Pull request merged**